cypress/pages/BounceRulesContainer/index.js

```diff
@@ -26,8 +30,94 @@ class BounceRulesPage extends Page {
     return cy.get(Selectors.ruleTable);
   }
 
-  open() {
-    super.open("/bounce_rules");
+  get deleteConfirmation() {
+    return cy.get(Selectors.deleteConfirmation);
+  }
+
+  get deleteConfirmationConfirm() {
+    return cy.get(Selectors.deleteConfirmationConfirm);
+  }
+
+  get deleteConfirmationAlert() {
+    return cy.get(Selectors.deleteConfirmationAlert);
+  }
+
+  get createRuleModal() {
+    return cy.get(Selectors.createRuleModal);
+  }
+
+  get priority() {
+    return cy.get(Selectors.priority);
+  }
+
+  get bounceAction() {
+    return cy.get(Selectors.bounceAction);
+  }
+
+  get responseCode() {
+    return cy.get(Selectors.responseCode);
+  }
+
+  get description() {
+    return cy.get(Selectors.description);
+  }
+
+  get enhancedCode() {
+    return cy.get(Selectors.enhancedCode);
+  }
+
+  get regex() {
+    return cy.get(Selectors.regex);
+  }
+
+  get submitButton() {
+    return cy.get(Selectors.submitButton);
+  }
+
+  get confirmModal() {
+    return cy.get(Selectors.confirmModal);
+  }
+
+  get confirmationSubmit() {
+    return cy.get(Selectors.confirmationSubmit);
+  }
+
+  get testRule() {
+    return cy.get("[data-rule='cypressTest']");
+  }
+
+  fillForm(
```

Comment:
See renaming comments and encapsulating all this into one createBounceRule function and the deletion steps into one deleteBounceRule function